### PR TITLE
Feature/display warning

### PR DESCRIPTION
**Pull request by kema91**
We want to display the warnings for each attribute difference.

### CODE REVIEW

File: src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java [hunk 1]
```diff
@@ -58,4 +66,15 @@ private String printBothDifferences( final AttributeDifference difference ) {
 		final Serializable actual = difference.getActual();
 		return String.format( KEY_EXPECTED_ACTUAL_FORMAT, key, expected, actual );
 	}
+
+	private String printWarning( final AttributeDifference difference ) {
+		final Integer warnings = difference.getElementIdentificationWarnings().size();
```
Comment (modulo11) [inline src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java]: Any reason to use a boxed integer?

File: src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java [hunk 2]
```diff
@@ -58,4 +66,15 @@ private String printBothDifferences( final AttributeDifference difference ) {
 		final Serializable actual = difference.getActual();
 		return String.format( KEY_EXPECTED_ACTUAL_FORMAT, key, expected, actual );
 	}
+
+	private String printWarning( final AttributeDifference difference ) {
```
Comment (modulo11) [inline src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java:70]: Why are `fileName` and `line` only extracted when there is only one warning?

File: src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java [hunk 3]
```diff
@@ -14,11 +15,18 @@
 
 	private static final String KEY_EXPECTED_ACTUAL_FORMAT = "%s: expected=\"%s\", actual=\"%s\"";
 
+	private static final String WARNING_FILENAME_LINE_FORMAT = ", warning=\"%s:%d\"";
```
Comment (modulo11) [inline src/main/java/de/retest/recheck/printer/AttributeDifferencePrinter.java]: Why is there a distinction between warning and warnings? What does warning mean for a user? Maybe there is a more expressive name (at least for displaying it).

**Comment by diba1013** [comment]
Since @kema91 is currently on vacation, I took over this PR to have it merged in the upcoming release.

I settled for the following changes:

1. Display all warnings from a `AttributeDifference`. However, currently, this should not happen as only one attribute can be used as identifying.
2. Rename "warning" to "breaks". What this term means cannot be deeply explained within the CLI output as it would clutter the message. Instead we should take care that this is properly documented.

**Review by modulo11 — APPROVED** [review]

**Pull request merged**